cli/cmd/testdata/install_default.golden

```diff
@@ -318,9 +304,7 @@ spec:
     spec:
       containers:
       - args:
-        - -addr=:8084
-        - -metrics-addr=:9994
-        - -api-addr=api:8085
+        - -api-addr=api.conduit.svc.cluster.local:8085
```

Comment:
When you merge this, there will be a conflict. When resolving the conflict, make sure you include the "." in front of the ":"  (i.e. ".") that is now on master.

Reply:
> When you merge this, there will be a conflict. When resolving the conflict, make sure you include the "." in front of the ":" (i.e. ".") that is now on master.

Sorry I didn't update this comment: I ended up having to back out the change that added the "." at the end of each domain name because @klingerf reported it broke stuff, and I haven't had time to debug the breakage. So please undo the appending of the "." to the domain names. Sorry again.